ext/hook/uhook.stub.php

```diff
@@ -75,6 +75,27 @@ public function overrideArguments(array $arguments): bool;
      */
     public function overrideReturnValue(mixed $value): bool;
 
+    /**
+     * Replaces the exception thrown by a function call. Must be called within a post-hook.
+     *
+     * @param \Throwable|null $exception An exception which will replace the original exception.
+     * @return bool 'true' on success, otherwise 'false'
+     */
+    public function overrideException(\Throwable|null $exception): bool;
+
+    /**
+     * Suppresses the call to the hooked function. Must be called within a pre-hook.
+     * @return bool always 'true'
+     */
+    public function suppressCall(): bool;
+
+    /**
+     * By default, advice is not called when the instrumented function from its advice.
```

Comment:
What is "advice"? I think you have the wrong english word.

Anyway, I suppose the goal is allowing the hook to run on a recursively called method?
Any reason to implement it through a method, instead of as a new flag on install_hook? Do we sometimes need to do that conditionally?